src/player.py

```diff
@@ -1,2 +1,12 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
+
+class Player:
+    def __init__(self, name, current_room):
+        self.name = name
+        self.current_room = current_room
+        self.items = list()
```

Comment:
Nice job here with the player class. `self.items` can also be set to a list literal `[]`